cou/apps/app.py

```diff
@@ -437,12 +437,17 @@ def upgrade_plan(self, target: OpenStackRelease) -> list[Optional[UpgradeStep]]:
     def post_upgrade_plan(self, target: OpenStackRelease) -> list[Optional[UpgradeStep]]:
         """Post Upgrade planning.
 
+        Wait until the application reaches the idle state and then check the target workload.
+
         :param target: OpenStack release as target to upgrade.
         :type target: OpenStackRelease
         :return: Plan that will add post upgrade as sub steps.
         :rtype: list[Optional[UpgradeStep]]
         """
-        return [self._get_reached_expected_target_plan(target)]
+        return [
+            self._get_wait_step(120),
```

Comment:
This should be defined as constant, but I was not sure about the right number. Maybe we can use same value for both model or only app.
I will not crate function, but rather constant only. Plus I do not see any benefit to hind logic for keystone, ceph-mon or rebitmq to `OpenStackApplication`, since we are defining new class for it.